internal/errors/code.go

```diff
@@ -45,6 +45,8 @@ const (
 	CycleFound                         = 121 // CycleFound represents an error when a cycle is found between a parent and child worker
 	WorkerConnNotFound                 = 122 // WorkerConnNotFound represents an error when a connection to a worker is not found
 	KmsWorkerUnsupportedOperation      = 123 // KmsWorkerUnsupportedOperation represents an error when a KMS worker is not supported for an operation
+	RetryLimitExceeded                 = 124 // RetryLimitExceeded represents an error when a retry limit is exceeded
+	QueueIsFull                        = 125 // QueueIsFull results in attempting to add an item to a queue which is full
```

Comment:
Where are these used?

Reply:
Could we add a comment? I think I see the practical reason for putting this here but it might still be confusing to see unused constants.